src/assets-build.ts

```diff
@@ -29,8 +29,10 @@ if (!fs.existsSync(projectDist)) {
 
 const manageCopy = (...copyargs: any[]) => copy(...copyargs)
   .on(
-    copy.events.COPY_FILE_COMPLETE,
-    (copyOperation: any) => console.info(`${copyOperation.src} => ${copyOperation.dest}`),
+    (copyOperation: any) => {
+      if (!process.argv[2]) return
```

Comment:
Maybe you can try to inject the augment instead of have this as a side effect (same for next parts)

Reply:
Okay, I will take a look!

Thanks for your quick review!